portfolio/src/main/webapp/charts/script.js

```diff
@@ -1,29 +1,73 @@
 google.charts.load('current', {'packages': ['corechart']});
-google.charts.setOnLoadCallback(drawChart);
+google.charts.setOnLoadCallback(drawStats);
 
-async function drawChart() {
+async function drawStats() {
   const response = await fetch('/pokedex-data');
-  const generationArray = await response.json();
-  const data = new google.visualization.DataTable();
+  const pokeData = await response.json();
+  const pokeStats = new google.visualization.DataTable();
+  pokeStats.addColumn('string', 'Stat Type');
+  pokeStats.addColumn('number', 'Min');
+  pokeStats.addColumn('number', '1st Quar');
+  pokeStats.addColumn('number', '3rd Quar');
+  pokeStats.addColumn('number', 'Max');
 
-  data.addColumn('string', 'Generation');
-  data.addColumn('number', 'Amount of Pokemon');
-  for (let i = 0; i < generationArray.length; i++) {
-    if (generationArray[i] !== 0) {
-      data.addRow([i.toString(), generationArray[i]]);
-    }
+  for (stat in pokeData) {
+    let quartiles = calculateQuartiles(pokeData[stat]);
+    pokeStats.addRow([stat, quartiles['min'], quartiles['firstQ'], quartiles['thirdQ'], quartiles['max']]);
   }
 
-  const options = {
-    'title': 'Pokemon Count per Generation',
-    'width': 500,
-    'height': 500,
+  var chart = new google.visualization.ComboChart(document.getElementById('chart-container'));
+  chart.draw(pokeStats, {
+    title : 'Box and Whisker Plot of Stats',
+    width: 600,
+    height: 400,
+    vAxis: {title: "Value"},
+    series: { 0: {type: "candlesticks"}}
+  });
+}
+
+function calculateQuartiles(arr) {
+  arr = removeNullsAndSort(arr);
+  let res = getMedian(arr);
+  let medianIdx = res['idx']
+  median = res['val'];
+  firstQ = getMedian(arr.slice(0,medianIdx))['val'];
+  thirdQ = getMedian(arr.slice(medianIdx))['val'];
+  return {
+    median: median,
```

Comment:
just a style thing, i _think_ in new versions of js if they key and the variable to store the values is the same, you can just do 

```
return {
  median,
  firstQ,
  thirdQ,
  min: something
}
```

might be worth seeing if this works.